app/_src/deck/guides/apiops.md

```diff
@@ -188,8 +188,24 @@ Most commonly, you will use the commands from CI/CD tools built into your versio
 to bring full and partial {{site.base_gateway}} configurations together to create APIOps for your 
 particular needs.
 
+## Extending OpenAPI Specifications
+Several custom annotations can be used within the OpenAPI specification allowing you to declare {{site.base_gateway}} capabilities directly in the specification document. All custom annotations related to {{site.base_gateway}} configuration are prepended with ```x-kong-``` label.
+
+- ```x-kong-tags```: specify the tags to use for each Kong entity generated. The tags can be overridden when doing the conversion. This can only be specified on document level.
```

Comment:
```suggestion
- `x-kong-tags`: Specify the tags to use for each {{site.base_gateway}} entity generated. Tags can be overridden when doing the conversion. This can only be specified at the document level.
```